tests/test_bgp_af.rb

```diff
@@ -106,6 +118,38 @@ def test_collection_not_empty
     end
   end
 
+  def config_ios_xr_dependencies(asn)
+    # These dependencies are required on ios xr
```

Comment:
@glennmatthews - I committed the rd auto property to feature/grpc last week - so '2' should work now, can look at latest global bgp @gvlaemyn .

Also @gvlaemyn don't forget to look at route-target which you will also inherit from the develop merge.